.github/workflows/regression-report.yml

```diff
@@ -122,24 +163,219 @@ jobs:
           # shellcheck disable=SC2005
           echo "$(cat "$GITHUB_OUTPUT")"
 
+
+  handle_config_bundle:
+    needs: [ parse_comment, checkout_and_cache ]
+    if: needs.parse_comment.outputs.mode == 'use_config_bundle'
+    runs-on: ubuntu-latest
+    outputs:
+      config_link: ${{ steps.set_links.outputs.config_link }}
+      projects_link: ${{ steps.set_links.outputs.projects_link }}
+    steps:
+      - name: Restore cached checkouts
+        uses: actions/cache@v4
+        with:
+          path: |
+            .
+            .ci-temp/checkstyle
+          key: ${{ runner.os }}-checkouts-${{ github.sha }}
+
+      - name: Set config and projects links
+        id: set_links
+        run: |
+          BASE_URL="https://raw.githubusercontent.com/checkstyle/test-configs/main"
+          CONFIG_BUNDLE="${{ needs.parse_comment.outputs.config_bundle }}"
+          CONFIG_LINK="$BASE_URL/$CONFIG_BUNDLE/config.xml"
+          PROJECTS_LINK="$BASE_URL/$CONFIG_BUNDLE/list-of-projects.properties"
+          ./.ci/append-to-github-output.sh "config_link" "$CONFIG_LINK"
+          ./.ci/append-to-github-output.sh "projects_link" "$PROJECTS_LINK"
+
+  handle_generate_config:
+    needs: [ parse_comment, checkout_and_cache ]
+    if: needs.parse_comment.outputs.mode == 'generate_config'
```

Comment:
`    if: needs.parse_comment.outputs.mode == 'generateed_config_bundle'`

Reply:
done

Reply:
this can be resolved.